examples/plotting/text.py

```diff
@@ -1,3 +1,13 @@
+''' A grid plot using descriptive text. This example demonstrates
+how to align text in a plain background in two dimensions. It also
+demonstrates how to change the angle of the texts in the plot.
```

Comment:
small edit suggestion:
```suggestion
demonstrates how to change the angle of text in a plot.
```